src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesEdit.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import { Card, Checkbox, Col, Layout, Row, Select, TextArea, TextField } from '@folio/stripes/components';
+import { required } from '../../../util/validators';
+
+export default class ExternalDataSourcesEdit extends React.Component {
+  static propTypes = {
+    actionButtons: PropTypes.func,
+    input: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      value: PropTypes.shape({
+        id: PropTypes.string,
+      }).isRequired,
+    }).isRequired,
+    meta: PropTypes.shape({
+      invalid: PropTypes.bool,
+      pristine: PropTypes.bool,
+      submitting: PropTypes.bool,
+    })
+  }
+
+  render() {
+    const { input: { value, name } } = this.props;
+    const kbadapterTypes = [{ value: 'org.olf.kb.adapters.EbscoKBAdapter', label: 'org.olf.kb.adapters.EbscoKBAdapter' }];
+    const recordTypes = [{ value: '1', label: 'Package' }];
```

Comment:
Can you just define these as member variables or constants? No need to recreate this array and object each render.